### PR TITLE
Interactive

### LINKED ISSUE #1749: Entity prompt doesn't appear in interactive training when there are no entities recognised
in interactive learning sometimes an intent is correctly identified, but includes an entity that is not annotated, only when the intent is marked as wrong, but then again is chosen as correct intent the user is asked to annotate the entity

### PR DESCRIPTION
**Pull request by paulaWesselmann**
**Proposed changes**:
- changed to ask the user to verify entities as well as intent in interactive learning, fixes #1749 

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
@paulaWesselmann is this different to your fork saving fix? looks like your commits from that are included here too

**Comment by paulaWesselmann** [comment]
@akelad yes, sorry I realized that too. The fork changes are already approved by Tom. the only thing else that changed here is the print statement for the user in interactive learning to ask for intent and entity.

File: rasa_core/training/interactive.py [hunk 1]
```diff
@@ -1005,6 +1005,7 @@ def _validate_user_text(latest_message: Dict[Text, Any],
     else:
         question = questionary.confirm(
             "Is the NLU classification for '{}' with intent '{}' correct?"
+            "\n    AND are all occurring entities labeled correctly?"
```
Comment (akelad) [inline rasa_core/training/interactive.py]: could we maybe rephrase this so it's a little more intuitive? Right now the sentences together don't make much sense. So, phrasing it like "Is the NLU classification for '{}': intent - {} and entities: {} correct?". Where entities would be none in the case of none of them

File: rasa_core/training/interactive.py [hunk 2]
```diff
@@ -997,15 +997,23 @@ def _validate_user_text(latest_message: Dict[Text, Any],
...
         print("The NLU classification for '{}' returned '{}'"
               "".format(text, intent))
         return False
     else:
         question = questionary.confirm(
-            "Is the NLU classification for '{}' with intent '{}' correct?"
-            "".format(text, intent))
+            "Is the NLU classification for '{}' with intent '{}' "
+            "and entities: {} correct?"
+            "".format(text, intent, entity))
```
Comment (akelad) [inline rasa_core/training/interactive.py]: ahh sorry about the back and forth on this, but this looks weird now:
` Is the NLU classification for 'i want a [greek](cuisine) restaurant' with intent 'inform' and entities: ['[greek](cuisine)'] correct?`

Can you come up with a better phrasing for this? maybe post it here and we can discuss before you change it again. Something like asking are all marked entities correct and the intent classification

Comment (paulaWesselmann) [reply rasa_core/training/interactive.py]: Do you think it is better to make this a two step question? first if the intent is classified correctly and then the entity? Or is that too many questions for the user?

Comment (akelad) [reply rasa_core/training/interactive.py]: hmmm i'm not sure which is better now tbh, could you post this in slack and ask for input? I feel like the two step thing might be too many times to hit enter

**Comment by paulaWesselmann** [comment]
I changed it to what we discussed in slack. It's ready for (hopefully) final review now.

File: rasa_core/training/interactive.py [hunk 3]
```diff
@@ -999,21 +999,20 @@ def _validate_user_text(latest_message: Dict[Text, Any],
     intent = parse_data.get("intent", {}).get("name")
     entities = parse_data.get("entities", [])
     if entities:
-        entity = []
-        for e in entities:
-            entity.append('[' + e['value'] + '](' + e['entity'] + ')')
+        msg = ("Is the intent '{}' correct for '{}' and are "
```
Comment (akelad) [inline rasa_core/training/interactive.py]: can we just rename `msg` to `message` everywhere?
